lib/csv_to_popolo.rb

```diff
@@ -335,7 +335,8 @@ def legislative_memberships
           area_id:         !r[:area_id].to_s.empty? ? r[:area_id] : !r[:area].to_s.empty? ? "area/#{_idify(r[:area])}" : nil,
           start_date:      r[:start_date],
           end_date:        r[:end_date],
-        }.select { |_, v| !v.nil? }
+          sources:         [{ url: r[:source] }],
+        }.reject { |_, v| v.nil? || v.empty? }
```

Comment:
I don't think this is going to work when `r[:source]` is `nil`. You'd end up with `sources` being set to `[{ url: nil }]`, which wouldn't get removed by the `reject` block.

Looks like this is why the validation tests from `bundle exec rake test:all` are failing on Travis.